js/main.js

```diff
@@ -24,7 +22,7 @@ function handleSubmit(event) {
   };
 
   data.nextEntryId++;
-  data.entries.push(entryValues);
+  data.entries.unshift(entryValues);
```

Comment:
changed to `unshift` method instead of `push`

Reply:
New entries will be the first index of the array

![image](https://github.com/t-doan/code-journal/assets/26643555/31ba97a4-7163-42b9-ac53-31de0f341dfa)